File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2022-05-15/examples-js/CosmosDBSqlDatabaseMigrateToManualThroughput.js
const { CosmosDBManagementClient } = require("@azure/arm-cosmosdb");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Migrate an Azure Cosmos DB SQL database from autoscale to manual throughput
 *
 * @summary Migrate an Azure Cosmos DB SQL database from autoscale to manual throughput
 * x-ms-original-file: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2022-05-15/examples/CosmosDBSqlDatabaseMigrateToManualThroughput.json
 */
async function cosmosDbSqlDatabaseMigrateToManualThroughput() {
  const subscriptionId = "subid";
  const resourceGroupName = "rg1";
  const accountName = "ddb1";
  const databaseName = "databaseName";
  const credential = new DefaultAzureCredential();
  const client = new CosmosDBManagementClient(credential, subscriptionId);
  const result = await client.sqlResources.beginMigrateSqlDatabaseToManualThroughputAndWait(
    resourceGroupName,
    accountName,
    databaseName
  );
  console.log(result);
}

cosmosDbSqlDatabaseMigrateToManualThroughput().catch(console.error);

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2022-08-15/examples-java/CosmosDBSqlDatabaseMigrateToManualThroughput.java
import com.azure.core.util.Context;

/** Samples for SqlResources MigrateSqlDatabaseToManualThroughput. */
public final class Main {
    /*
     * x-ms-original-file: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2022-08-15/examples/CosmosDBSqlDatabaseMigrateToManualThroughput.json
     */
    /**
     * Sample code: CosmosDBSqlDatabaseMigrateToManualThroughput.
     *
     * @param azure The entry point for accessing resource management APIs in Azure.
     */
    public static void cosmosDBSqlDatabaseMigrateToManualThroughput(
        com.azure.resourcemanager.AzureResourceManager azure) {
        azure
            .cosmosDBAccounts()
            .manager()
            .serviceClient()
            .getSqlResources()
            .migrateSqlDatabaseToManualThroughput("rg1", "ddb1", "databaseName", Context.NONE);
    }
}
